core/relay/Makefile

```diff
@@ -1,6 +1,6 @@
-DOCKER_IMAGE_NAME ?= dlt-interop/relay
-DOCKER_TAG ?= latest
-DOCKER_REGISTRY ?= res-dlt-interop-docker-local.na.artifactory.swg-devops.com
+DOCKER_IMAGE_NAME ?= weaver-relay
```

Comment:
I'd prefer using standard tagging terminology here, like `hyperledger-labs/weaver-dlt-interoperability/relay`. That way the organization of Docker images built on one's machine is easy to comprehend.

Reply:
Marking it resolved after conversation on webex.